documentation/api/commands.markdown

```diff
@@ -90,9 +90,23 @@ payload of this command.
 
 ### "deactivate node", version 2
 
-The payload is expected to be the certname of a node, as a raw JSON
-string, which will be deactivated effective as of the time the command
-is *processed*.
+The payload is a JSON map, formatted as follows:
+
+     {
+      "certname": <string>,
+      "producer_timestamp": <datetime>
+     }
```

Comment:
Well, its all about creating obvious navigation I guess, right now our other commands have their own documents and this one doesn't (largely for historical reasons, wire formats use to mean something else, ie. they were related to queries also). Also, the formats are different, 

But beyond just creating this new document, lots of other things bother me about this document, the separation of data related to the command seems to be in two places, this higher level doc, and the individual wire format docs. I'd like to see a more concerted effort to push the per command data into their own docs over time, in particular the historical blurb we add here about differences to the last revision, and even examples (which we have at the bottom of the document). I also want to see the term "wire format" replaced with something more related to commands, since I feel like wire format is more of an internal term these days, and it might not make sense to consumers. Anyway.